### PR TITLE
Techdebt/folder renaming

### PR DESCRIPTION
**Pull request by samuelWilliams99**
Change folder structure to be:
`content/models/...` -> `content/models/rvr/...`
`content/materials/...` -> `content/materials/rvr/...`

Also changes auto include to include all resources

### CODE REVIEW

**Review by plally — APPROVED** [review]

File: gamemode/load_assets.lua [hunk 1]
```diff
@@ -0,0 +1,16 @@
+local function runRecurse( dir, ext, f )
+    local files, dirs = file.Find( dir .. "/*", "GAME" )
+    if not files then return end
```
Comment (overlordbleck) [inline gamemode/load_assets.lua:9]: `file.Find` returns nil if the path is not valid. Does an empty table count as a falsey value?

**Review by overlordbleck — COMMENTED** [review]
💢

File: gamemode/load_assets.lua (hunk 1 above)
Comment (samuelWilliams99) [reply gamemode/load_assets.lua:9]: it does not, only nil and false are falsey in lua

File: gamemode/load_assets.lua [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    for k, v in pairs( files ) do
```
Comment (overlordbleck) [inline gamemode/load_assets.lua]: Can we use a more descriptive variable name for `v`

File: gamemode/load_assets.lua [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+        if string.match( v, "^.+%." .. ext .. "$" ) then
+            f( dir .. "/" .. v )
+        end
+    end
+    for k, v in pairs( dirs ) do
```
Comment (overlordbleck) [inline gamemode/load_assets.lua]: ^

File: gamemode/load_assets.lua (hunk 2 above)
Comment (samuelWilliams99) [reply gamemode/load_assets.lua]: aye, ill clean this func up a lil

**Comment by samuelWilliams99** [comment]
Confirmed working

**Pull request merged**